apps/docs-website-fumadocs/src/components/magicui/rainbow-button.tsx

```diff
@@ -0,0 +1,31 @@
+import { cn } from "@/lib/utils";
+import React from "react";
+
+interface RainbowButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {}
+
+export const RainbowButton = React.forwardRef<
```

Comment:
[`forwardRef` is deprecated in React 19](https://react.dev/blog/2024/12/05/react-19#ref-as-a-prop)

Reply:
> [`forwardRef` is deprecated in React 19](https://react.dev/blog/2024/12/05/react-19#ref-as-a-prop)

I didn't write this myself, some "shadcn cli" wrote this.
It works, so its fine for now.